### DISCUSSION TITLE
Welcome to harbor Discussions!

### DISCUSSION THREAD

**Discussion by AnthonyRonning** [HarborWallet/harbor#64] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chrisBonGit** [comment]
Hi
This looks really promising, thanks for simplifying ecash management!

I tried using mutiny to generate an invite to a federation (freedom one), but despite the preview working ok, when I attempted to add freedom one I was given the error that this is not a valid invitation.

Paste from nix terminal;
ERROR harbor::fedimint_client   > Fedimint on different network bitcoin, expected: signet
 ERROR harbor::core              > Error adding federation: Network mismatch, expected: signet
 INFO  harbor::fedimint_client   > initializing a new federation client: c944b2fd1e7fe04ca87f9a57d7894cb69116cec6264cb52faa71228f4ec54cd6

**Reply by benthecarman** [reply]
Harbor doesn't work for mainnet federations yet. You can try a signet one

faucet.mutinynet.com

**Reply by chrisBonGit** [reply]
Oh - I've got ahead of my skis, ok. Thanks. 
I tried to use that signet federation but now harbor hangs after I enter my password.
I used the just reset-db but still see the following in the nix terminal;

ERROR harbor::fedimint_client > Fedimint on different network bitcoin, expected: signet
thread 'tokio-runtime-worker' panicked at src/core.rs:494:26:
Could not create fedimint client: Network mismatch, expected: signet
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

Any suggestions? 
Note, after resetting the database I first only used 'just release', but then also tried building. Both resulted in the same error being thrown.
